components/create_comment/create_comment.jsx

```diff
@@ -697,7 +703,12 @@ class CreateComment extends React.PureComponent {
 
         const {draft} = this.state;
         const updatedDraft = {...draft, message};
-        this.props.onUpdateCommentDraft(updatedDraft);
+
+        cancelAnimationFrame(this.saveDraftFrame);
+        this.saveDraftFrame = requestAnimationFrame(() => {
```

Comment:
Why `requestAnimationFrame` and not `setTimeout`?

Reply:
We discussed different ways of throttling this, but we settled on requestAnimationFrame since it only throttles when the app is lagging and it matches what we're doing in the post textbox already. If we weren't using that, we'd probably go for something like lodash's throttle/debounce methods that we use elsewhere